ingester/tests/test_ingester.py

```diff
@@ -22,66 +25,68 @@ def maker():
     return maker
 
 
+@mock.patch.dict(os.environ, {"AWS_REGION":"us-east-1"})
```

Comment:
Can you use pytest's `monkeypatch` facility instead? Should be `monkeypatch.setenv("AWS_REGION", "us-east-1")`

Reply:
Thanks, didn't know about monkeypatch

Fixed in [58ceef1](https://github.com/planetlabs/datalake/pull/78/commits/58ceef10364b8b918c670b84da66bbd9882d3175)